src/olympia/users/templates/users/edit.html

```diff
@@ -15,9 +15,15 @@
 {% block content %}
 {% include 'users/includes/navigation.html' %}
 <div id="user_edit" class="primary prettyform grid" role="main">
+  {% if switch_is_active('fxa-auth') and user.has_anonymous_display_name() %}
+    <div class="notification-box error anonymous">
+      <h2>{{ _('Please set your display name') }}</h2>
+      <p>{{ _('Please set your display name or username to complete the registration process.') }}</p>
```

Comment:
maybe it should be:

"Please set your display name ... when completing the registration process"

?

Because it's not required.